packages/kit/src/runtime/server/utils.js

```diff
@@ -52,7 +53,13 @@ export function allowed_methods(mod) {
  * @param {string} message
  */
 export function static_error_page(options, status, message) {
-	return text(options.templates.error({ status, message }), {
+	let page = options.templates.error({ status, message });
+
+	if (DEV) {
+		page = page.replace('</head>', '<script type="module" src="/@vite/client"></script></head>');
```

Comment:
We should comment what this does so we don't get confused later

Reply:
```suggestion
		// inject Vite HMR client, for easier debugging
		page = page.replace('</head>', '<script type="module" src="/@vite/client"></script></head>');
```